docs/4-routing.md

```diff
@@ -1 +1,161 @@
-# Routing
\ No newline at end of file
+# Routing
+
+Hubro uses file-based routing inside the `/pages` directory.
+Each directory in `pages` corresponds to a URL and can hold
+either:
+
+- [Pages](#pages)
+- [Actions](#actions)
+- [API routes](#api-routes)
+
+## Pages
+
+Hubro's file-based router looks for `page.js` files inside each subdirectory of `/pages`.
+
+- `/pages`
+  - `/about`
+    - `page.js`
+  - `page.js`
+
+Pages must include a function as its default export. That function must return
+a Lit `TemplateResult` (typically [html](https://lit.dev/docs/api/static-html/#html)).
```

Comment:
The examples I could find all did this, but I'm not sure if it's strictly required to return a `TemplateResult` here.